tracing/opentelemetry/src/main/java/io/helidon/tracing/opentelemetry/OpenTelemetrySpan.java

```diff
@@ -118,6 +118,7 @@ public Optional<String> baggage(String key) {
     // Check if OTEL Context is already available in Global Helidon Context.
     // If not – use Current context.
     private static Context getContext() {
+        Context.current().makeCurrent();
```

Comment:
Not an expert in the API, but this line give me pause. We are making current something that is current? And the scope that is returned (which presumably needs closing at some point) is ignored? Can we explain this better?